temporal-sdk/src/test/java/io/temporal/client/functional/UpdateTest.java

```diff
@@ -102,6 +99,44 @@ public void updateCompletedWorkflowUntyped() {
     assertThrows(ExecutionException.class, () -> updateRef.getResultAsync().get());
   }
 
+  @Test
+  public void updateWorkflowDuplicateId() {
+    Assume.assumeTrue(
+        "skipping for real server because the real server does not handle duplicate update ID correctly",
```

Comment:
Can we get a `TODO` + a link to a GH issue where they will solve it?

Reply:
I would like to, but the issue is tracked in Jira. I can link to the PR itself if you want https://github.com/temporalio/temporal/pull/4297?

Reply:
Ah, so long as tracked, no problem.